person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+
+  attr_accessor :age, :name
+
+  def initialize(id, age, name = 'unknown', parent_permission: true)
+    @id = id
```

Comment:
@sam2if, id shouldn't be passed. It should be generated randomly.

E.G:
` @id = Random.rand(1..1000)`